src/ui/flutter_app/lib/shared/theme/app_theme.dart

```diff
@@ -1,104 +1,66 @@
+/*
+  This file is part of Sanmill.
+  Copyright (C) 2019-2021 The Sanmill developers (see AUTHORS file)
+
+  Sanmill is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Sanmill is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
 import 'package:flutter/material.dart';
 import 'package:sanmill/services/storage/storage.dart';
-import 'package:sanmill/shared/constants.dart';
 import 'package:sanmill/shared/theme/colors.dart';
 
+/// The Apps Theme
+@immutable
 class AppTheme {
   const AppTheme._();
-  // TODO: restructure theming. Some theme Elements should be accessed via Theme.of(context)
 
-  // Theme data
+  // TODO: [Leptopoda] when using a base theme the darkMode is somehow broken ¿?
```

Comment:
¿ -- Is this a punctuation mark in Spanish?

Reply:
Yep Spanish does use the "upside down question mark" for questions. Although they use the upside down one at the beginning of a sentence and the normal one at the end. like "¿Everything ok?" 

I developed a habit to use this annotation for hard questions I can't find answers to. As said above I don't think our todo comments need to follow a specific style guide

Reply:
I just feel curious, I have never seen this kind of punctuation.  I am not familiar with other European languages.